Rdmp.Core/DataViewing/ViewTableInfoExtractUICollection.cs

```diff
@@ -89,9 +89,6 @@ public string GetSql()
             if (filter != null)
                 qb.RootFilterContainer = new SpontaneouslyInventedFilterContainer(memoryRepository, null, new[] { filter }, FilterContainerOperation.AND);
 
-            if (ViewType == ViewType.Aggregate)
```

Comment:
I removed this because there is a throw on line 94/97 anyway so this statement is pointless.  Probably its there because it was copied from the ColumnExtractionCollection code